tesseract-core/src/schema.rs

```diff
@@ -138,10 +139,13 @@ impl From<SchemaConfigJson> for Schema {
                     .collect()
             });
 
+        let default_locale_str: &str = "en";
```

Comment:
can we make this a `const DEFAULT_LOCALE_STR` and put it at the top of the schema? (that's my preference, anyways, let me know if you prefer this way of handling hardcoded stuff)

Reply:
@hwchen agreed! I've just updated this